apps/website/package.json

```
@@ -70,6 +68,8 @@
     "storybook-addon-next-router": "4.0.2",
     "style-dictionary": "3.7.2",
     "style-loader": "3.3.1",
+    "tailwind-scrollbar": "2.1.0",
+    "tailwindcss": "3.2.7",
     "tw-colors": "^1.1.6"
```

これと `pnpm-lock.yaml` 再生成してちょ 🙏
```suggestion
    "tw-colors": "1.1.6"
```